styles.css

```diff
@@ -0,0 +1,53 @@
+body {
+    background-color: #1F1926;
+    margin: 0;
+    padding:0;
+    min-height: 100vh;
```

Comment:
I understand @maribelpreite wanted the mininum height to cover the whole browser windows. Scrollbar can sometimes happen with 100vh because some browsers have additional bars etc that are not included in the calculation. Maybe using 100% would rather help? It's less definitive.

Reply:
Yes, I wanted it to use as much space as possible without needing the scrollbar. max-height instead of min-height makes more sense to me, however it doesn't work either. 100% value doesn't work neither for min-height, nor for max-height. I don't know what that can be, becase the rest of the units are responsive. 

What I've also noticed is that if the window broser gets shrinked, the elements keep their original size, they do not adjust to the size of the window. So it's not acting responsively, even though the size units are responsive. 

I would also like the number of columns to act responsively, right now it's set to 4 because I know there are 12 cards and they'll look nicely distributed. But if I have 10 cards, then I'd rather have 3 columns and 4 rows. I guess I have to write some logic in JS to calculate the number of columns and then give it back to CSS in grid-template-columns property. Any suggestions to help me find the way?